python-lib/dku_deeplearning_image/dku_constants.py

```diff
@@ -37,6 +37,28 @@
 COMPILE_LOSS_FUNCTION = 'categorical_crossentropy'
 COMPILE_METRICS = ["accuracy"]
 
+# Options for custom form
+GPU_MEMORY_LIMIT = "memory_limit"
+GPU_MEMORY_GROWTH = "memory_growth"
+
+POOLING_OPTIONS = [
```

Comment:
Use enums, that's the right data structure for what you want to do

Reply:
Will do in a new PR